source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/PIMS S61.00 Change Log.txt

```diff
@@ -6,6 +7,12 @@ PSP_PIMS S61.00 Sprint 60 2023-Aug-11
   - PIMS_ACQUISITION_FILE_HIST
   - PIMS_COMPENSATION_REQUISITION
   - PIMS_COMPENSATION_REQUISITION_HIST
+========
+PSP_PIMS S60.01 Sprint 61 2023-Aug-24
+- Added alter scripts:
+  - 079_DML_PIMS_PROPERTY_ROAD_TYPE_Alter_Down.sql
+  - 079_DML_PIMS_PROPERTY_ROAD_TYPE_Alter_Up.sql
+>>>>>>>> upstream/test:source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/PIMS S60.01 Change Log.txt
```

Comment:
@devinleighsmith did you fix conflicts on this pr? this one seems to have been left unhandled...